department-of-veterans-affairs/va.gov-cms
# VACMS-15503 Add multi workflow filters to all Views that have exposed content moderation filter

## Description

Relates to #15503

## Testing done


## Screenshots


## QA steps

What needs to be checked to prove this works?
What needs to be checked to prove it didn't break any related things?
What variations of circumstances (users, actions, values) need to be checked?

As an admin go to the following Views and make sure the Moderation state filter is **present** and **works**.
- [x] Go to  https://pr15769-4wq0cjp3zdblysiz1xjp3l1xc9kegtfq.ci.cms.va.gov/admin/content/audit/phone-numbers, admin/content/audit/buttons
- [x] Go to   https://pr15769-4wq0cjp3zdblysiz1xjp3l1xc9kegtfq.ci.cms.va.gov/admin/content/audit/metatag
- [x] Go to   https://pr15769-4wq0cjp3zdblysiz1xjp3l1xc9kegtfq.ci.cms.va.gov/admin/content/audit/knowledge-base-article-administration
- [x] Go to   https://pr15769-4wq0cjp3zdblysiz1xjp3l1xc9kegtfq.ci.cms.va.gov/admin/content/audit
- [x] Go to   https://pr15769-4wq0cjp3zdblysiz1xjp3l1xc9kegtfq.ci.cms.va.gov/admin/content/bulk
- [x] Go to   https://pr15769-4wq0cjp3zdblysiz1xjp3l1xc9kegtfq.ci.cms.va.gov/admin/content/events
- [x] Go to   https://pr15769-4wq0cjp3zdblysiz1xjp3l1xc9kegtfq.ci.cms.va.gov/admin/content/resources-and-support
- [x] Go to   https://pr15769-4wq0cjp3zdblysiz1xjp3l1xc9kegtfq.ci.cms.va.gov/admin/content


### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
